code/src/pages/MoviesDetails.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from "react-router-dom"
+import { Link } from "react-router-dom"
+
+import { apiKey } from "key"
+import "./moviesDetails.css"
+import Arrow from "../assets/arrow.svg"
+
+export const MoviesDetails = () => {
+    const { movieId } = useParams();
+    const [movies, setMovies] = useState([])
+    const [loading, setLoading] = useState(true)
+
+    useEffect(() => {
+        const movieDetailsURL = `https://api.themoviedb.org/3/movie/${movieId}?api_key=${apiKey}&language=en-US`
+        fetch(movieDetailsURL)
+            .then((res) => {
+                return res.json()
+            })
+            .then((json) => {
+                console.log(json)
+                setMovies(json)
+                setLoading(false)
+            })
+            .catch((error) => {
+                console.error('Bad request', error)
+            })
+    }, [movieId, loading])
+
+    if (!movies.id) {
+        return (
+
+            <div className="not-found">
+                {loading && <div className="loader" />}
+                {}
+                <h3>Movie not found, probably because you searched for a non existing one.</h3>
+                <h4>But check out the movies we do have, just go back.</h4>
+                <Link to="/">
+                    <p className="error-message">Go back home</p>
+                </Link>
+            </div>
+        )
+    }
+
+    return (
+
+        <div className="movie" style={{ backgroundImage: `linear-gradient(rgba(0, 0, 0, 0) 10%, rgb(0, 0, 0) 100%), url(https://image.tmdb.org/t/p/w1280${movies.backdrop_path})` }}>
+            <Link className="back-link" to="/" exact='true'>
```

Comment:
There is limited set of props available for `<Link />` predefined component provided by `react-router-dom` package and `exact` is not one of them - basically, it's not doing anything as it's not available for it. It's only available for `<Route />` and `<NavLink />` components.

For further reference check out [https://reactrouter.com/web/api/Link](https://reactrouter.com/web/api/Link)